### PR TITLE
Temporary side nav

### PR DESCRIPTION
**Pull request by Naseer-007**
Created a temp side nav with our doc pages so we can access them, with translated routes for the component pages

### CODE REVIEW

**Review by alexandreGrenierIRCC — APPROVED** [review]

File: component-library/src/app/shell/shell.component.ts [hunk 1]
```diff
@@ -92,6 +92,99 @@ export class ShellComponent implements OnInit {
     iconTrailing: 'fa-regular fa-arrow-right',
     header: true
   };
+
+  overviewPage: INavigationItemLink = {
+    id: 'accordionNav',
```
Comment (alexandreGrenierIRCC) [inline component-library/src/app/shell/shell.component.ts:97]: ID's need to be unique

**Review by alexandreGrenierIRCC — CHANGES_REQUESTED** [review]

File: component-library/src/assets/locales/en.json [hunk 2]
```diff
@@ -863,13 +870,53 @@
...
+		"Title": "Icon Button",
 		"Intro": "Icon buttons can be used for a single, well-understood action that can be clearly communicated with a stand-alone icon.",
-		"InteractiveDemoDesc": "Try our icon button component and the configurable options available within it.",
+		"InteractiveDemoDesc": "Try our <strong>icon button</strong> component and the configurable options available within it.",
 		"PrimaryText": "Primary icon buttons are used to highlight important and easy-to-understand actions. It can be used in a group of icon buttons.",
-		"CriticalText": "Critical icon buttons are often used for destructive actions like deleting user-contributed content."
+		"CriticalText": "Critical icon buttons are often used for destructive actions like deleting user-contributed content.",
+		"OrganizeIcons": "Organize icon button groups",
+		"OrganizeIconsAlt": "A document viewer tool bar with pagination icons, zoom icons and resize icons grouped and separeted.",
+		"OrganizeIconsDoText": "Logically organize icon buttons that have a contextual relationship by keeping related actions together and unrelated actions separated either by white space or dividers.",
```
Comment (kristopher-houston) [inline component-library/src/assets/locales/en.json:885]: Is this all linting?

Comment (Naseer-007) [reply component-library/src/assets/locales/en.json:885]: These are coming from the google doc

File: component-library/src/app/shell/shell.component.ts (hunk 1 above)
Comment (Naseer-007) [reply component-library/src/app/shell/shell.component.ts]: Fixed

**Review by alexandreGrenierIRCC — APPROVED** [review]

**Review by kristopher-houston — APPROVED** [review]

**Pull request merged**